### PR TITLE
Set up ci

### PR DESCRIPTION
**Pull request by AnasNaouchi**
## Description

Add the required github actions workflows to test and publish the package.

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -73,7 +73,7 @@ class _MyHomePageState extends State<MyHomePage> {
       MaterialPageRoute(
           builder: (context) => omisePayment.authorizePayment(
                 authorizeUri: Uri.parse(
-                    "https://yourAuthUri"), // Replace with actual authorization URL
+                    "https://3dsms.staging-omise.co/payments/pay2_61mtfxeqe8e5zbdilyn/authorize"), // Replace with actual authorization URL
```
Comment (new4762) [inline example/lib/main.dart:76]: Can we remove `staging`part from example?, I don't think we should show this endpoint for public repo

Comment (AnasNaouchi) [reply example/lib/main.dart:76]: yeah my bad, let me change...

File: test/services/omise_payment_service_test.dart [hunk 2]
```diff
@@ -4,6 +4,7 @@ import 'package:omise_flutter/src/pages/select_payment_method_page.dart';
 import 'package:omise_flutter/src/pages/payment_authorization_page.dart';
 import 'package:omise_flutter/omise_flutter.dart';
 import 'package:webview_flutter/webview_flutter.dart';
+// ignore: depend_on_referenced_packages
```
Comment (new4762) [inline test/services/omise_payment_service_test.dart:7]: Just wonder  this is safe to ignore right?

**Review by new4762 — APPROVED** [review]

File: test/services/omise_payment_service_test.dart (hunk 2 above)
Comment (AnasNaouchi) [reply test/services/omise_payment_service_test.dart:7]: This is just to ignore the warning as I am using some internal class to verify my unit tests and this class is not exposed by the package that i am using directly.

Comment (new4762) [reply test/services/omise_payment_service_test.dart:7]: 📓 Noted

**Review by aashishgurung — APPROVED** [review]

**Pull request merged**